### ISSUE TITLE
Connecting pins

### ISSUE THREAD

**Issue by Multisoft1978** [ExpressLRS/ExpressLRS-Hardware#30]
Hello
I don't understand how it's connected.
Can I connect it to Taranis Q X7?
TX - SBUS/PPM ?
3.3V
GND
RX -> S-PORT?

**Comment by kkbin505** [comment]
This is an receiver (RX) not TX, you can not connect it to transmitter. You can only connect it to a FC.

**Comment by mariusdp** [comment]
If you mean diy TX to QX7, you connect only 3 wires. V+, GND and SPORT.

**Comment by Multisoft1978** [comment]
Yes, I already figured it out.  Now me another question torments, why does a 500mW module in a DIY project work at a maximum of 250mW?  :)

**Comment by mariusdp** [comment]
has to do with stability/performance. That is the max power output ELRS supports for now...

**Comment by OpenUAS** [comment]
The assumption "why does a 500mW module in a DIY project work at a maximum of 250mW?" is incorrect, having a 1000mW module in one of our DIY projects works at a maximum of 1004mW. @Multisoft1978 Having answered the second question, as asked in this in non related base issue #30, which is not a ELRS Hardware issue per se, IMHO now is the time to close this issue.
